WeScan/Review/ReviewViewController.swift

```diff
@@ -9,11 +9,13 @@
 import UIKit
 
 /// The `ReviewViewController` offers an interface to review the image after it has been cropped and deskwed according to the passed in quadrilateral.
-final class ReviewViewController: UIViewController {
+open class ReviewViewController: UIViewController {
     
     private var rotationAngle = Measurement<UnitAngle>(value: 0, unit: .degrees)
-    private var enhancedImageIsAvailable = false
-    private var isCurrentlyDisplayingEnhancedImage = false
+    open var enhancedImageIsAvailable = false
+    open var isCurrentlyDisplayingEnhancedImage = false
+    open var enhanceButtonTintColorOn: UIColor = .white
```

Comment:
I feel that the `: UIColor` part in this case follows a different format to the rest of the project.

Could we use `open var enhanceButtonTintColorOn = UIColor.white` instead?